core/lexicon/en/default.inc.php

```diff
@@ -143,6 +143,9 @@
 $_lang['error'] = 'Error';
 $_lang['error_sending_email'] = 'Error sending email';
 $_lang['error_sending_email_to'] = 'Error while sending mail to ';
+$_lang['error_grid_get_content_no_msg'] = 'A server error prevented this grid’s content from loading. To get additional information, change MODx’s debug system setting to either 1 or empty (not 0, as this will turn reporting off).';
```

Comment:
```suggestion
$_lang['error_grid_get_content_no_msg'] = 'A server error prevented this grid’s content from loading. To get additional information, change MODX’s debug system setting to either 1 or empty (not 0, as this will turn reporting off).';
```

Reply:
Critical tweak.